packages/testlab/test/testlab.smoke.js

```diff
@@ -1,8 +1,8 @@
-const should = require('should');
-const testlab = require('..');
+const assert = require('assert');
 
 describe('testlab smoke test', () => {
   it('exports expect interface', () => {
-    testlab.expect.should.be.type('function');
+    const testlab = require('..');
+    assert.equal(typeof testlab.expect, 'function');
   });
 });
```

Comment:
@bajtos Do you know how to disable should/as-function so that I can require `../lib-es2015` in another test? I'm getting conflict issues.

Try it out yourself and let me know if you have any ideas -- Ideally, I want to export do this in `index.js`:

```
const testlab = module.exports = require('./lib');
testlab.es2015 = testlab.es6 = require('./lib-es2015');
```

# Error

```
...
TypeError: Cannot redefine property: to
```